docs/Documentation/Applications/LBMcfd.md

```diff
@@ -0,0 +1,124 @@
+
+Its meshfree nature makes it very convenient to handle and resolve complex geomtries such as cracks and porous microstructures. The algorithmically simple nature of the LBM which consists of hopping of particles by a pre-determined distance followed by a local update for time increment makes the solver trivial to implement on a GPU, which results into very fast and scalable solvers that can be used as a data generators in a machine learning pipeline. 
+
+
+![alt text](LBMcfd_metadata/lbmAlgo.png)
+
+The LBM is a recast of fluid dynamics into a fully discrete kinetic system for the populations $f_i(\mathbf{x},t)$ of particles, which are associated with the discrete velocities $\mathbf{c}_i$ fitting into a regular space-filling lattice. As a result, the kinetic equations for the populations $f_i(\mathbf{x},t)$ follow a simple algorithm of `"stream along links $\mathbf{c}_i$ and collide at the nodes $\mathbf{x}$ in discrete time $t$". The method computes a discrete version of the Boltzmann transport equation, which mathematically describes the state of the fluid with a Gaussian distribution in the velocity space with its mean representing the local fluid velocity and the variance representing the local energy of the fluid. The dynamics of the fluid then evolve with a streaming-relaxation equation for these probability distribution functions. The probablistic nature of the method makes it a gateway to quantum computing for CFD. 
+
+
+
+## Overview
+
+At NREL, serveral packages are available for the purpose, each with their strengths, pros and cons. The matrix below provides a birds eye view of the available packages. (All company, product and service names used on this page are for identification purposes only. Use of these names, trademarks and brands does not imply endorsement.)
+
+|                                                                              | Windows| Mac OS  | Linux (HPC) | CPU    | GPU    | Cost | Speciality                   |
+|:----------------------------------------------------------------------------:|:------:|:-------:|:-----------:|:------:|:------:|:----:|:----------------------------:|
+| [M-Star](https://mstarcfd.com/)                                              | y      | x       | y           | y      | y      | $    | GUI, moving geometries       |
+| [NREL marbles](https://nrel.github.io/marbles/VandV.html)                         | y      | y       | y           | y      | y      | Free | Open source, compressible  |  
+| [Ansys Discovery](https://www.ansys.com/products/3d-design/ansys-discovery)  | y      | x       | x           | x      | y      | $    | GUI, geometry modeling       |
+
+Only M-Star and marbles can be run on the Kestrel HPC system. Users with an access to GPU enabled Windows computers or Virtual Machines may try Ansys Discovery at their own discretion.
```

Comment:
Since Ansys Discovery isn't available on our systems, I don't think we should include it in the documentation. The documentation is intended to be how to do things on HPC, but not necessarily explaining scientific content in general. Message me if you want to discuss more.